ecs-cli/modules/cli/compose/project/project_parseV3_test.go

```diff
@@ -94,11 +140,123 @@ services:
 	project.ecsContext.ComposeFiles = append(project.ecsContext.ComposeFiles, tmpfile1.Name())
 	project.ecsContext.ComposeFiles = append(project.ecsContext.ComposeFiles, tmpfile2.Name())
 
-	// assert number of container configs matches expected number of services
-	// TODO: assert containerConfig content after conversion implemented
+	// get map of docker-parsed ServiceConfigs
+	expectedConfigs := getServiceConfigMap(t, project.ecsContext.ComposeFiles)
+
+	// assert # and content of container configs matches expected services
 	conConfigResult, err := project.parseV3()
 	if err != nil {
 		t.Fatal("Unexpected error parsing file: ", err)
 	}
 	assert.Equal(t, numOfServices, len(*conConfigResult))
+	for _, conConfig := range *conConfigResult {
+		verifyConvertToContainerConfigOutput(t, expectedConfigs[conConfig.Name], conConfig)
+	}
+}
+
+func verifyConvertToContainerConfigOutput(t *testing.T, observed types.ServiceConfig, expected containerconfig.ContainerConfig) {
+
+	// verify equivalent fields
+	assert.Equal(t, expected.CapAdd, observed.CapAdd)
```

Comment:
Would it be helpful to add an error message saying which field failed? (Not sure if a failure would already do that in a readable way)